api/interservice/authz/v2/project.proto

```diff
@@ -76,6 +76,7 @@ service Projects {
     rpc ListRulesForProject(ListRulesForProjectReq) returns (ListRulesForProjectResp) {};
     rpc DeleteRule(DeleteRuleReq) returns (DeleteRuleResp) {};
     rpc ListRulesForAllProjects(ListRulesForAllProjectsReq) returns (ListRulesForAllProjectsResp) {};
+    rpc ApplyRules(ApplyRulesReq) returns (ApplyRulesResp) {};
```

Comment:
Hmmm, on second though, should this go up in line 69 and be called `ProjectUpdateApply`? Seems like it goes with those APIs.